psl-core/src/main/java/org/linqs/psl/application/inference/InferenceApplication.java

```diff
@@ -31,6 +32,8 @@
 import org.linqs.psl.reasoner.term.TermGenerator;
 import org.linqs.psl.reasoner.term.TermStore;
 import org.linqs.psl.util.Reflection;
+import org.slf4j.Logger;
```

Comment:
Third-party imports should be its own section: This Project (PSL) -> Third-party -> Standard Library.